backend/tests/test_main.py

```
@@ -0,0 +1,128 @@
+    mod.__path__ = []
+    try:
+        from importlib.machinery import ModuleSpec
+        mod.__spec__ = ModuleSpec(name, loader=None)
+    except Exception:
+        mod.__spec__ = None
+    return mod
+
+# Stub heavy optional dependencies to avoid import errors
+tf = stub_module('tensorflow')
+tf.keras = stub_module('tensorflow.keras')
+tf.keras.models = stub_module('tensorflow.keras.models')
```

Pylint (3.3.7)

[error] Instance of 'module' has no 'keras' member

(E1101)